_includes/codepens/custom-toolbar-group-button/index.js

```diff
@@ -0,0 +1,22 @@
+tinymce.init({
+  selector: 'textarea#custom-toolbar-group-button',
+  height: 500,
+  toolbar_mode: 'floating',
+  toolbar: 'alignment',
+
+  content_css: [
+    '//fonts.googleapis.com/css?family=Lato:300,300i,400,400i',
+    '//www.tiny.cloud/css/codepen.min.css'
+  ],
+
+  setup: function (editor) {
+
+    /* example, adding a group toolbar button */
+    editor.ui.registry.addGroupButton('alignment', {
```

Comment:
This should be `addGroupToolbarButton` right? Or should we have renamed the function to `addGroupButton` in bridge?
(Same in a few other places in this PR)

Reply:
Good catch, thanks @SimonFc! :+1:

Reply:
Should be all fixed now :smiley: